docs/var.tfvars-doc.md

```diff
@@ -60,7 +60,12 @@ In order to retrieve the PowerVS region, zone and instance specific details plea
    ibmcloud_zone       = eu-de-1
    service_instance_id = 360a5df8-3f00-44b2-bd9f-d9a51fe53de6
    ```
+### Fips Variable for openshift deployment
```

Comment:
Move this down.

Reply:
Please move this down to section https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/docs/var.tfvars-doc.md#openshift-installation-details with proper formatting.